README.md

```
@@ -256,6 +256,7 @@ Any provider package registers under the `caspian.providers` entry-point group.
 | <img src="https://cdn.simpleicons.org/telegram" width="14"/> &nbsp;Telegram (bot) | ✅ | ✅ |
 | <img src="https://cdn.simpleicons.org/discord" width="14"/> &nbsp;Discord | ✅ | ✅ one-click |
 | <img src="https://cdn.simpleicons.org/slack" width="14"/> &nbsp;Slack | ✅ | ✅ one-click |
+| <img src="https://cdn.simpleicons.org/bluesky" width="14"/> &nbsp;Bluesky | ✅ | ✅ instant |
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)